core/client/src/test/java/alluxio/client/file/FileOutStreamTest.java

```diff
@@ -156,6 +165,9 @@ public void flush() {
     };
     mUnderStorageFlushed = underStorageFlushed;
 
+    when(mFactory.create(any(InetSocketAddress.class), anyLong())).thenReturn(
+        mUnderStorageOutputStream);
+
     // Set up underFileStorage so that we can test UnderStorageType.SYNC_PERSIST
     mUnderFileSystem = ClientMockUtils.mockUnderFileSystem();
     when(mUnderFileSystem.create(anyString())).thenReturn(mUnderStorageOutputStream);
```

Comment:
we never run this test with delegation disabled, so these seem like dead lines of code that should be deleted.